recipes/bmx/all/conanfile.py

```diff
@@ -0,0 +1,131 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, copy, get, rm, rmdir
+import os
+
+required_conan_version = ">=1.53.0"
+
+
+class BmxConan(ConanFile):
+    name = "bmx"
+    description = (
+        "Library for handling broadcast/production oriented media file formats. "
+        "Allows reading, modifying and writing media metadata and file essences."
+    )
+    topics = ("vfx", "image", "picture", "video", "multimedia", "mxf")
+    license = "BSD-3-Clause"
+    homepage = "https://github.com/bbc/bmx"
+    url = "https://github.com/conan-io/conan-center-index"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_libcurl": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_libcurl": True,
+    }
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+    def requirements(self):
+        # Required libraries
+        self.requires("uriparser/0.9.8")
+        self.requires("expat/2.6.2")
+
+        if not (self.settings.os == 'Windows' or self.settings.os == 'Macos'):
+            self.requires('libuuid/1.0.3')
+
+        # Configuration dependent requirements
+        if self.options.with_libcurl:
+            self.requires("libcurl/8.6.0")
+
+    def validate(self):
+        if self.settings.compiler.cppstd:
+            check_min_cppstd(self, 11)
```

Comment:
We probably also want to comment out https://github.com/bbc/bmx/blob/main/CMakeLists.txt#L37 so that users get to properly choose the standard version used :)

Reply:
Done. Good catch. This showed me another problem - which I added a second patch for and prepare the upstream PR now. (Will add the link to the PR when I have it ready.)